circusweb/circushttpd.py

```diff
@@ -121,7 +123,7 @@ def render_template(self, template_path, **data):
             template = app.loader.load(template_path)
             return template.generate(**namespace)
         except Exception:
-            print exceptions.text_error_template().render()
+            six.print_(exceptions.text_error_template().render())
```

Comment:
Can't we use ``from __future__ import print_function`` and use ``print`` instead ?